src/pages/workspaces/workspace/Data.js

```diff
@@ -146,8 +154,9 @@ class WorkspaceData extends Component {
   }
 
   componentDidUpdate() {
-    const { workspaceEntities, selectedEntityType, itemsPerPage, pageNumber } = this.state
-    StateHistory.update({ workspaceEntities, selectedEntityType, itemsPerPage, pageNumber })
+    const { workspaceEntities, selectedEntityType, selectedEntities, itemsPerPage, pageNumber } = this.state
+
```

Comment:
Any reason for the extra line here each time? Seems the usage is tightly-contained in most cases.